src/controllers/shared/common.resume.submission.controller.ts

```diff
@@ -110,5 +111,15 @@ const setWebApplicationData = (session: Session, appData: ApplicationData, trans
     mapTrustApiReturnModelToWebModel(appData);
   }
 
+  if (isActiveFeature(config.FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC) && (!appData.entity_number)) {
+    console.debug("Removing old NOCs");
+    appData[BeneficialOwnerIndividualKey] = (appData[BeneficialOwnerIndividualKey] as BeneficialOwnerIndividual[])
```

Comment:
it might be simpler to delete them directly rather than loop through each one and just return the element

delete appData[BeneficialOwnerIndividualKey][NonLegalFirmNoc]
delete appData[BeneficialOwnerOtherKey][NonLegalFirmNoc]
delete appData[BeneficialOwnerGovKey][NonLegalFirmNoc]

Reply:
👍 Will see if that works and implement.

Reply:
BeneficialOwner... are array
correction of the syntax in my comment with Martin